src/components/Drawer.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styles from '../stylesheets/styles'
+
+const _ = require('lodash')
+
+class Drawer extends Component {
+  static propTypes = {
+    cells: PropTypes.shape({}).isRequired,
+    label: PropTypes.string.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      showDetails: false
+    }
+  }
+
+  toggleDetails(e) {
+    e.preventDefault()
+
+    this.setState({
+      showDetails: !this.state.showDetails
+    })
+  }
+
+  render() {
+    const {
+      cells,
+      label
+    } = this.props
+
+    return (
+      <a
+        href="{undefined}"
```

Comment:
@yih-en I have never seen this before. Is it different from `javascript:void(0)`?